hmac_authentication/hmacauth.py

```diff
@@ -1,57 +1,65 @@
 # -*- coding: utf-8 -*-
 
 from hmac_authentication import exceptions
+
 import base64
+import collections
+import enum
```

Comment:
I like enum (obviously, since I suggested it), but it's only part of the standard library in python3. For this to work in python2 (which is good practice, even though we mostly use 3.4 at 18f), we can add the backported package to `install_requires` in setup.py.